Log/v1.5_Changes.md

```diff
@@ -0,0 +1,51 @@
+# Version 1.5 Changes
+
+For many many reasons, we're making some significant changes to the program that will go into effect starting June 19, 2017.
+
+This document outlines the planned changes from a high level. More details to follow.
+
+## What's going away
+
+- Stats (i.e. most of what the "game" formerly was)
+- Levels
+- Wandering in the wilderness (lack of learning tracks)
+
+## What's staying
+
+- Project-based learning (at later phases)
+- Apprenticeship (at later phases)
+- Focus on skill-building
+- Free coffee
+
+## New stuff
+
+- Phases (5 of them, designed to be completed in sequence)
+- More support (appropriate to the phase)
+- Phase assessments
+- New membership review process based on assessments
+- Tighter scaffolding & structure at earlier phases
+- More career prep & support in final phase
+
+## Where to find more information
+
+To learn about Phases and Assessments, check out the new [Phases section](/Phases).
+
+More information will be shared as it becomes available. If you have questions in the meantime, ask a Moderator.
+
+## The Transition
+
+Moving from a loosely sequenced program to a more structured one is not a simple process. How can we know which learners should be in which phase?
+
+To solve this problem, during the week of 6/19 _everyone_ will take one of the three [Phase Assessments](/Phases/Assessments). Based on the outcome of this assessment, learners will be placed into either Phase 1, 2, 3, 4, or 5.
+
+We're calling this the **"Sorting Assessment"** because its purpose is to _sort_ learners into the best phase for their stage of learning. Since stats weren't a reliable indicator of level & skill, we're essentially "re-leveling" everyone through a more systematic, objective, and fair process.
+
+If you pass the assigned sorting assessment, you'll go into the following phase (e.g. if you pass the phase 2 assessment, you'll be placed into phase 3). If you fail the sorting assesssment, you'll be assigned to the phase assessed (e.g. if you fail the phase 2 assessment, you'll be assigned to phase 2).
```

Comment:
This isn't quite right.

If you fail the assessment that you choose, you're placed into the previous phase and have up to 8 weeks to try and pass it again.